src/equiv.js

```diff
@@ -126,6 +126,48 @@ QUnit.equiv = (function() {
 					parentsB.pop();
 					return true;
 				},
+				
+				"set": function( b, a ) {
+					var aArray, bArray;
+					
+					// b could be an object literal here
```

Comment:
actually b could be any other kind of object.

This might be removed, its clear we are checking both arguments are both instances of Set.